### ISSUE TITLE
Can't create a second Driver after destroying the first one

### ISSUE THREAD

**Issue by LiamPietralla** [zwave-js/ZWaveJS.NET#48]
After calling `driver.Destroy()` I would expect to be able to create and start a new driver instance.

Instead the `DriverReady` event is never fired, and I can see this is because the `Controller` property is never set.

Not sure if something needs to be updated, so that the driver and be re-created and started again.

**Comment by marcus-j-davies** [comment]
HI @LiamPietralla - Thanks for the report.

I believe this has already been fixed in V4.
The `Destroy` method is now a lot more destructive in v4.

See:
https://github.com/zwave-js/ZWaveJS.NET/blob/0d4a2894fec1cfde07e79af77b203b43829b5f1d/Visual%20Studio%20Projects/ZWaveJS.NET/ZWaveJS.NET/Driver.cs#L535

V4 is not far from being released.

**Comment by LiamPietralla** [comment]
Awesome thanks @marcus-j-davies, will check that out later

**Comment by marcus-j-davies** [comment]
I am in the middle of fine tuning V4 before release, so I wouldn't jump right in yet.

But see the following PR's to gain a view on the changes to expect.

 - https://github.com/zwave-js/ZWaveJS.NET/pull/31
 - https://github.com/zwave-js/ZWaveJS.NET/pull/47

**Comment by marcus-j-davies** [comment]
@LiamPietralla 

iff you want to test V4 - pull down the code, and also pulldown the Latest PSI  - can be found in the V4 branch (if using the embedded runtime)

**Comment by LiamPietralla** [comment]
Yeah interesting, I am getting "The Platform Support Image version (server.psi) is lower than the requested schema version" even though I am using the library in Client mode, do I need to include the new psi in the build maybe? Will take another look tomorrow

**Comment by marcus-j-davies** [comment]
Hi @LiamPietralla  - no you don't need to (I need to change the message)

if in client mode, it will default to using the schema it was built against, you will need set the schema of the server you are connecting to with the over load.

**Comment by marcus-j-davies** [comment]
Message Changed:
https://github.com/zwave-js/ZWaveJS.NET/pull/31/commits/7e5b75829377b420227b89d834a8e09949c9503a

**Comment by LiamPietralla** [comment]
@marcus-j-davies awesome, all working now and no issues with V4! Much thanks 😀
